lib/web_console/templates/_inner_console_markup.html.erb

```diff
@@ -3,6 +3,8 @@
   <div class='console-actions'>
     <div class='close-button button' title='close'>x</div>
   </div>
-  <div class='console-inner'></div>
+  <div class='console-inner'>
+    <div class='console-notifications'></div>
```

Comment:
Do we really need the extra layout here? What's so special about the notifications, they look like a regular console line to me.

Reply:
Initially I had the ```console-notifications``` set to be updated whenever the binding changes, but it will disappear out of the scrolling window when the commands exceeds the scrollable view. 

I have changed it to append before the lastChild of the ```prompt-box``` so that it remains in view, but I forgot to remove the unused ```console-notifications``` div. It is now removed!